hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeDescriptor.java

```diff
@@ -678,6 +682,16 @@ public void addBlockToBeReplicated(Block block,
     replicateBlocks.offer(new BlockTargetPair(block, targets));
   }
 
+  /**
+   * Store ec block to be replicated work.
+   */
+  @VisibleForTesting
+  public void addECBlockToBeReplicated(Block block,
+                                     DatanodeStorageInfo[] targets) {
```

Comment:
You can write it in one line. If you are using IntelliJ, you can set Hadoop code style using `dev-support/code-formatter/hadoop_idea_formatter.xml`.
```suggestion
  public void addECBlockToBeReplicated(Block block, DatanodeStorageInfo[] targets) {
```

Reply:
Thanks for your advice, it's useful to me.